frontend/frames/settings.py

```diff
@@ -8,16 +11,23 @@ class SettingsWindow():
     def __init__(self, window, base, **kwargs):
         self.window = window
         self.base = base
+        self.config = self.read_config()
+        self.name = StringVar()
+        self.name.trace('w', self.track_changes)
```

Comment:
I'm not a fun of having Tkinter and config file functionality in one class here. Can we create separate file for this `INI` file, that will expose object, that will return, save settings like geting, setting item in dictionary?

Reply:
See:
https://docs.python.org/3.6/reference/datamodel.html#object.__getitem__
https://docs.python.org/3.6/reference/datamodel.html#object.__setitem__